tests/test_invenio_communities.py

```diff
@@ -284,34 +284,39 @@ def assert_header_links(response, ref, page, size):
 
 def test_communities_rest_get_details(app, db, communities):
     """Test the OAI-PMH Sets creation."""
+    (comm1, comm2, comm3) = communities
     with app.test_client() as client:
-        response = client.get('/api/communities/comm1')
+        response = client.get('/api/communities/{}'.format(str(comm1.id)))
```

Comment:
Communities API is not part of Zenodo documentation at the moment, however we might want to add it at some point. However, Deposit API is already using community names (I don't think we would want to change that), hence we might want to stick with names instead of UUIDs across all of our APIs relating to communities.